src/frontend/src/index.ts

```diff
@@ -48,11 +49,23 @@ const printVersion = () => {
   if (version.dirty) {
     console.warn("This version is dirty");
   }
+
+  if (anyFeatures()) {
+    const featuresRec: Record<string, boolean> = features;
+    const message = `
+Some features are enabled:
+${Object.keys(featuresRec)
```

Comment:
nth: would `Object.entries` fits better?

```
Object.entries(features).map(([key, value]) => ` - ${key}: ${value}`)
```

Reply:
it would! thanks!